plugins/docker/docker_

```diff
@@ -54,35 +55,60 @@ from multiprocessing import Process, Queue
 
 
 def print_containers_status(client):
-    running = 0
-    paused = 0
-    created = 0
-    restarting = 0
-    removing = 0
-    exited = 0
-    dead = 0
-    for container in client.containers.list():
+    running = []
+    paused = []
+    created = []
+    restarting = []
+    removing = []
+    exited = []
+    dead = []
+    for container in client.containers.list(all=True):
```

Comment:
Maybe `sorted()` around the container list would make sure, that the list of hosts always uses the same order?

Reply:
I ended up doing it in the caching wrapper, using a decorator.